### PR TITLE
Feature/multiple tables

### PR DESCRIPTION
**Pull request by Felipe-Perez-Ferraro**
- Created owners and species table
- Inserted information to owners and species table
- Added queries to owners and species table

## Write queries (using JOIN) to answer the following questions

- What animals belong to Melody Pond?

- List of all animals that are pokemon (their type is Pokemon).

- List all owners and their animals, remember to include those that don't own any animal.

- How many animals are there per species?

- List all Digimon owned by Jennifer Orwell.

- List all animals owned by Dean Winchester that haven't tried to escape.

- Who owns the most animals?

### CODE REVIEW

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Felipe-Perez-Ferraro, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a Readme file with a `Live Demo` section for a project without a live link. Kindly remove the `Live Demo` section as it is irrelevant to this kind of project. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: schema.sql [hunk 1]
```diff
@@ -6,3 +6,14 @@ CREATE TABLE animals (
...
+CREATE TABLE owners (
+  id SERIAL PRIMARY KEY,
+  full_name VARCHAR(30),
+  age INTEGER
+);
+
+CREATE TABLE species (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(30)
+);
```
Comment (Munsa1) [inline schema.sql:21]: - [ ] It's obvious you did alter the `Animals` table, but as required you need to document the changes made here.

e.g.

```
   ALTER TABLE animals DROP column species;
   ALTER TABLE animals ADD species_id int REFERENCES species(id);
   ALTER TABLE animals ADD owner_id int REFERENCES owners(id);
```

File: data.sql [hunk 2]
```diff
@@ -14,4 +14,17 @@ VALUES
...
+	('Jennifer Orwell', 19),
+	('Bob', 45),
+	('Melody Pond', 77),
+	('Dean Winchester', 14),
+	('Jodie Whittaker', 38);
+
+INSERT INTO species(name)
+VALUES
+	('Pokemon'),
+	('Digimon');
```
Comment (Munsa1) [inline data.sql]: - [ ] Again here we need to document the code that executes that if an animal's name ends with `mon` it should have `Digimon` as a `species_id`, all other animals should have `Pokemon` instead. Kindly do so

e.g
```
   UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Digimon')
   WHERE name like '%mon';

   UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Pokemon')
   WHERE species_id IS NULL;

```

Comment (Munsa1) [inline data.sql]: - [ ] Laqstly here in `data.sql` we need to document the code that executes that all animals got assigned owners. Checkout the following example for the first one.

e.g
```
   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Sam Smith')
   WHERE name = 'Agumon';
```

File: README.md [hunk 3]
```diff
@@ -48,14 +47,6 @@ This project is a table where you cand find animals and their features
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Munsa1) [inline README.md:48]: Kindly fix the following few issues with the readme file

1. Clicking on the `Deployment` link in the table of contents should take the user to the deployment part of the `Getting Started` section, kindly fix this broken link.
image

2. Kindly fix the broken link to the issues page under the `Contributing` section for a more professional readme file.
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Felipe-Perez-Ferraro , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Added SceenShots :heavy_check_mark: 
Correct ScreenShot Results :heavy_check_mark: 
Descriptive PR :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbayomiOlaoye — APPROVED** [review]
## STATUS: APPROVED :green_circle:  

Hi @Felipe-Perez-Ferraro   :wave: :wave:, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Cleanly written code :white_check_mark: 
- Update done as requested by reviewer ✅ 
- Nicely implemented requirements ✅ 
- Great repo documentation ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**